packages/DeadCode/src/Rector/Plus/RemoveZeroAndOneBinaryRector.php

```diff
@@ -105,9 +105,11 @@ private function processBinaryPlusAndMinus(BinaryOp $binaryOp): ?Expr
      */
     private function processBinaryMulAndDiv(BinaryOp $binaryOp): ?Expr
     {
-        if ($this->isValue($binaryOp->left, 1)) {
-            if ($this->isNumberType($binaryOp->right)) {
-                return $binaryOp->right;
+        if ($binaryOp instanceof Mul) {
```

Comment:
This will skip all the `Div`, right?

Or will it change:

```diff
-5/1
+5
```

Reply:
it will only skip the div for 1/5:
$binaryOp->left == 1 
$binaryOp->right == 5

Reply:
Could you add test case for it, just to be sure?